.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Usually it is not a good practice to commit your IDE configuration files inside the project. Different collaborators use different IDEs and different configurations. You do not want to track the changes of such files.